src/portal/src/lib/components/replication/replication.component.ts

```diff
@@ -82,6 +82,10 @@ export class SearchOption {
   pageSize: number = DEFAULT_PAGE_SIZE;
 }
 
+const STATUS_MAP = {
+  "Succeed": "Succeeded"
```

Comment:
Could you put the constant STATUS_MAP in some golable variable file or language json file?  We needn't define it twice in this file and replication-tasks.component.ts.

Reply:
In fact , it's a back-end api bug.  So modify where it appears